packages/typo3-guides-cli/src/Command/MigrateSettingsCommand.php

```diff
@@ -0,0 +1,306 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\GuidesCli\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class MigrateSettingsCommand extends Command
+{
+    protected static $defaultName = 'typo3-guides';
```

Comment:
Class is final, so use private please ;-) Also in the methods below.

Reply:
Changed this, where it worked. Note that $defaultName MUST be protected:

```
PHP Fatal error:  Access level to T3Docs\GuidesCli\Command\MigrateSettingsCommand::$defaultName must be protected (as in class Symfony\Component\Console\Command\Command) or weaker in /Users/garvin/TYPO3/render-guides/packages/typo3-guides-cli/src/Command/MigrateSettingsCommand.php on line 13
```